cluster.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Cluster
```

Comment:
Is this used in github actions?

I guess it's based on what I did for local cluster testing?

Reply:
Yes, I will remove this in the next push.